File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/DTOs/SubcategoryDTO.cs
namespace SafarnamaApplication.DTOs
{
    public class SubcategoryDTO
    {
        public int SubcategoryId { get; set; }
        public string SubcategoryCode { get; set; }
        public int? CategoryId { get; set; }
        public string? SubcategoryName { get; set; }
        public string? SubcategoryImagePath { get; set; }
        public bool? Flag { get; set; }
    }
}

File: Java_Backend/E-Tour/src/main/java/dto/SubcategoryDTO.java
package dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class SubcategoryDTO {
	 	private Integer subcategoryId;
	    private String subcategoryName;
	    private String subcategoryImagePath;
	    private Boolean flag;
	    private Integer categoryId; // avoid nesting whole Category entity
}
